wp-content/themes/csisjti/assets/_scss/style.scss

```diff
@@ -11,6 +11,8 @@
 @use 'components/media';
 @use 'components/event-block';
 @use 'components/entry-header';
+@use 'components/filters-modal';
```

Comment:
I would not include the filters-modal or modal-grid as a part of `style.scss`, but rather `resource-library.scss` since they're only used on that page.